### PR TITLE
feat(@angular/cli): adding script for stress app

### PR DESCRIPTION
**Pull request by sumitarora**
Fixes: https://github.com/angular/angular-cli/issues/5921

Stress App: https://github.com/sumitarora/angular-stress-tester

### CODE REVIEW

File: scripts/stress-app.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Promise.resolve()
+  .then(() => console.log('Cloning application...'))
+  .then(() => execute(`git clone "https://github.com/sumitarora/angular-stress-tester" "${outputPath}"`))
+  .then(() => console.log('Moving into Directory...'))
+  .then(() => process.chdir(outputPath))
+  .then(() => console.log(`Current Directory: ${process.cwd()}`))
+  .then(() => console.log('Installing...'))
+  .then(() => execute('npm install'))
+  .then(() => console.log('Building...'))
```
Comment (hansl) [inline scripts/stress-app.js]: The goal of such a big app is to time the build step, so maybe insert some performance metric gathering here and show it at the end? We really don't care about the previous steps' performance.

**Comment by hansl** [comment]
Also, looking at the stress app, it seems like there's no lazy routes and no dependencies between components. I  think these 2 would affect performance quite a lot. Could you add that?

File: scripts/stress-app.js (hunk 1 above)
Comment (sumitarora) [inline scripts/stress-app.js]: @hansl added metrics for usage

**Comment by sumitarora** [comment]
@hansl All Changes have been done. I have added some metrics in Script and added dependencies in the components.

**Comment by filipesilva** [comment]
@hansl can you re-review?

**Comment by hansl** [comment]
Closing this PR in favor of just using the github repo for the app.

**Pull request closed without merging**